### DISCUSSION TITLE
[IDEA] List Feature

### DISCUSSION THREAD

**Discussion by hilmysyarif** [kirimwa-id/forum#15] [Saran dan Ide]
Halo Mas Rio,
Dibawah ini mungkin ada sedikit ide dan saran dari saya, 

- API remaining Quota
- Auto Text Config API (untuk mentrigger respon ketika ada chat masuk menggunakan kata2 tertentu)
- Support Emoticon Message
- Support untuk meng-create kontak di kirimWA dengan kategori / tag
- Support untuk mereplace ##kode## dengan text yang sudah di set (cth: No INV. ##kode## maka akan tampil di message WA : No INV. 12345)

Terima kasih.

**Comment by rioastamal** [comment]
Halo mas @hilmysyarif,

Terima kasih sarannya.

> API Remaining Quota 

Sudah ada pada end point `GET /v1/quotas` cek pada dokumentasi https://developer.kirimwa.id/api/v1/#quotas

> Auto Text Config API

Fitur ini bisa diimplementasikan tapi pada sisi aplikasi anda ketika fitur Webhook Pesan Masuk #7 

> Support Emoticon Message

Seharusnya mengirim emoji dengan karakter UTF-8 bisa contohnya 😎 . Apa ada masalah?

> Support untuk meng-create kontak di kirimWA dengan kategori / tag

Ini mungkin perlu diperjelas lagi. Apakah artinya kontak ini berbeda dengan kontak di perangkat.

> Support untuk mereplace ##kode## dengan text yang sudah di set (cth: No INV. ##kode## maka akan tampil di message WA : No INV. 12345)

Sama seperti usulan fitur Auto Text Config API, fitur ini terlalu spesifik jika diterapkan di API. Dan bisa diterapkan disisi aplikasi jika fitur Webhook pesan masuk #7 sudah dirilis.
